### PR TITLE
http2: disable priority forwarding

### PR DESCRIPTION
**Pull request by Kriechi**

### CODE REVIEW

**Comment by Kriechi** [comment]
Too complex? It's two if statements...
The logging is set to `debug`, not very verbose IMO...

File: mitmproxy/options.py [hunk 1]
```diff
@@ -65,15 +66,18 @@ def __init__(
...
         ciphers_server: Optional[str]=None,
         clientcerts: Optional[str] = None,
-        http2: bool = True,
         ignore_hosts: Sequence[str] = [],
         listen_host: str = "",
         listen_port: int = LISTEN_PORT,
         upstream_bind_address: str = "",
         mode: str = "regular",
         no_upstream_cert: bool = False,
-        rawtcp: bool = False,
```
Comment (mhils) [inline mitmproxy/options.py]: I think this was sorted alphabetically before.

File: mitmproxy/tools/cmdline.py [hunk 2]
```diff
@@ -137,16 +137,19 @@ def get_common_options(args):
         ciphers_client = args.ciphers_client,
         ciphers_server = args.ciphers_server,
         clientcerts = args.clientcerts,
-        http2 = args.http2,
```
Comment (mhils) [inline mitmproxy/tools/cmdline.py]: alphabetically as well

File: mitmproxy/options.py (hunk 1 above)
Comment (Kriechi) [inline mitmproxy/options.py]: Nope - should I make it sorted?

File: mitmproxy/tools/cmdline.py (hunk 2 above)
Comment (Kriechi) [inline mitmproxy/tools/cmdline.py]: Nope - again...

**Pull request merged**